src/styles/service-modify.module.css

```diff
@@ -84,6 +84,11 @@
     vertical-align: middle;
 }
 
+.modify-history-request-content {
```

Comment:
actually this content is there for all orders.. not just modify. 
Its better we remove the word modify from everywhere.